chapters/en/_toctree.yml

```
@@ -3,7 +3,7 @@
   - local: chapter0/1
```

```suggestion
  - local: chapter0/introduction
```
and maybe we can rename the file too!